### ISSUE TITLE
Any plans to publish this to Pypi?

### ISSUE THREAD

**Issue by point-source** [Crewscope/html2bbcode#1]
I am trying to revive an old project and it depends on the original version of this lib. I would like to change the dependency over to point towards this version but I don't see it on pypi. Any plans to upload it there?

**Comment by ehsanf** [comment]
I don't have permission to publish this under the same name on PyPi. You can directly pip install from a github project. Alternatively I can rename the package and publish on Pypi with a different naming. That will mean that you will have to update dependencies and imports on your dependent projects. Let me know ...

**Comment by point-source** [comment]
Makes sense. I just pointed it at the repo for now. Mostly I just want to make sure it will remain available. Stuff on pypi is more permanent than github.
